TFT/src/User/API/config.c

```diff
@@ -903,15 +903,13 @@ void parseConfigKey(uint16_t index)
     #ifdef FIL_RUNOUT_PIN
       case C_INDEX_RUNOUT:
         if (inLimit(config_int(), 0, 3))
-          infoSettings.runout = config_int();
+          SET_BIT_VALUE(infoSettings.runout, RUNOUT_ENABLED, GET_BIT(config_int(), RUNOUT_ENABLED));
```

Comment:
do you need { } after "if"?

Reply:
Yes, I need it. Thanks for providing it. 😉